doc/developers/contributing.rst

```diff
@@ -726,6 +726,16 @@ We are glad to accept any sort of documentation:
 
 .. dropdown:: Guidelines for writing docstrings
 
+  * You can use `pytest` to test docstrings, e.g. assuming the
+    `RandomForestClassifier` docstring has been modified, the following command
+    would test it's docstring compliance:
+
+    .. prompt:: bash
+
+      pytest --doctest-modules sklearn/ensemble/_forest.py -k RandomForestClassifier
+
+  * The correct order of sections is: Parameters, Returns, See Also, Notes, Examples
```

Comment:
I would instead redirect to this section from the `numpydoc` documentation: https://numpydoc.readthedocs.io/en/latest/format.html#sections